src/snowflake/cli/plugins/snowpark/venv.py

```diff
@@ -13,7 +13,6 @@
 
 from snowflake.cli.plugins.snowpark.models import (
     Requirement,
-    RequirementType,
```

Comment:
Probably we can delete this class altogether

Reply:
Good catch, removed